frontend/projects/valtimo/components/src/lib/components/page-header/page-header.component.scss

```
@@ -51,10 +51,19 @@
   }
+
+  &:not(.page-head--title-as-breadcrumb) valtimo-page-actions {
+    flex: 1 1 auto;
+    width: auto;
+    min-width: 0;
+  }
 }
 
 .page-header-container {
   width: min-content;
+  // Allow this flex item to shrink below min-content so a long title truncates
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)